static_src/components/user_list.jsx

```diff
@@ -125,16 +129,22 @@ export default class UserList extends React.Component {
   }
 
   render() {
+    let saving;
     let loading = <Loading text="Loading users" />;
     let content = <div>{ loading }</div>;
 
+    if (this.props.saving) {
```

Comment:
Also I get a propTypes warning for `loadingDelayMS`, we can just pass `{0}` as a number.